Makefile

```diff
@@ -24,6 +24,10 @@ GO ?= go
 GOFLAGS := 
 # Set to 1 to use static linking for all builds (including tests).
 STATIC := $(STATIC)
+# The cockroach image to be used for starting Docker containers
+# during acceptance tests. Usually cockroachdb/cockroach{,-dev}
+# depending on the context.
+COCKROACH_IMAGE :=
```

Comment:
I thought that was a gnu make only thing so I went for vanilla. Kind of nice to see where it's being used, too. Have no strong feelings either way though so feel free to clean it up next time you touch that stuff.